### ISSUE TITLE
Question about the accurate values of the training loss and the surface error

### ISSUE THREAD

**Issue by Zhanyuan23333** [nathanrgodwin/overfit-shapes#5]
I want to do some comparative experiments with your results, but i can only find a histogram without concrete number in your paper. Could you provide some data used to draw the histogram to me ? Thank you.

**Comment by nathanrgodwin** [comment]
@Zhanyuan23333 The paper is not actually mine, this is just my implementation of the paper. For information about the data in the paper you'll have to reach out to the original authors.
